### PR TITLE
Improve display methods of (graded) rows, columns and FreydCategoryObjects/Morphisms

### PR DESCRIPTION
**Pull request by HereAround**
- As suggested by @mohamed-barakat , general infomation on an object/ a morphism is printed at the end of the output.
- Minor changes in the formatting.

### CODE REVIEW

File: FreydCategoriesForCAP/gap/FreydCategory.gi [hunk 1]
```diff
@@ -1537,22 +1537,27 @@ InstallMethod( Display,
...
     
-    Print( Concatenation( "\n\n", "Range:\n" ) );
+    Print( "Range:\n" );
+    Print( "--------------------------------\n\n" );
     Display( RelationMorphism( Range( freyd_category_morphism ) ) );
-    Print( "--------------------------------" );
-
+    Print( Concatenation( "\n\n", "--------------------------------", "\n" ) );
+    
+    Print( "General data:\n" );
```
Comment (sebastianpos) [inline FreydCategoriesForCAP/gap/FreydCategory.gi]:
```suggestion
    Print( "General description:\n" );
```
I would rather like to have a clean distinction between this and the term Morphism *datum*.

File: FreydCategoriesForCAP/gap/FreydCategory.gi [hunk 2]
```diff
@@ -1561,12 +1566,16 @@ InstallMethod( Display,
...
-
-    Print( Concatenation( "\n\n", "Relation morphism:\n" ) );
     
+    Print( Concatenation( "\n", "--------------------------------\n" ) );
+    Print( "Relation morphism:\n" );
+    Print( "--------------------------------\n\n" );
     Display( RelationMorphism( freyd_category_object ) );
+    Print( Concatenation( "\n\n", "--------------------------------", "\n" ) );
+    
+    Print( "General data:\n" );
```
Comment (sebastianpos) [inline FreydCategoriesForCAP/gap/FreydCategory.gi]:
```suggestion
    Print( "General description:\n" );
```

**Comment by HereAround** [comment]
Thanks for suggesting this. This is perfectly fine for me.

**Comment by sebastianpos** [comment]
Thank you.

**Pull request merged**